src/components/PBSCPasswordStrength.js

```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+const strengthListDictionary = [
+  {
+    strength: 'EMPTY',
+    color: ['#C4C4C4', '#C4C4C4', '#C4C4C4', '#C4C4C4', '#C4C4C4'],
```

Comment:
not sure to understand, why do we have five colors here? is it a color for every strength?

Reply:
I have refactor it. Each strength have a color. Everything after this level of strength is grey. Everything before that level of strength follow that color pattern :  ['#DC2619', '#FFA114', '#76BC21', '#349724', '#287C1A']

So if the strength level is 3, the color will be : '#DC2619', '#FFA114', '#76BC21', '#C4C4C4', '#C4C4C4,

Grey = #C4C4C4